### DISCUSSION TITLE
Build failed for nvidia-display-driver

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/meta-tegra#1229] [Q&A]
Hi. 

When I tried to build `nvidia-display-driver`, it returned a lot of 

> aarch64-poky-linux-gcc: error: unrecognized command line option '-mno-outline-atomics'; did you mean '-fno-inline-atomics'

I searched and found that this option required gcc 9.4.0 or higher. Is my gcc recipe in poky outdated, or the gcc on my ubuntu host needs updating?

Thanks in advance.

**Comment by ichergui** [comment]
Hey @CHNtentes 
Interesting!
Are you using [tegra-demo-distro](https://github.com/OE4T/tegra-demo-distro) ? if so, which branch ?

**Comment by madisongh** [comment]
>  Is my gcc recipe in poky outdated, or the gcc on my ubuntu host needs updating?

It's the cross-compiler that's issuing the complaint. If you're on the kirkstone branch or later of poky, you should have gcc 11 or later, unless you, or one of the layers you have added to your build, have changed things to use a different compiler/version.
